docs/build/differences-vs-evm/index.md

```diff
@@ -172,3 +186,24 @@ If you’re already familiar with blockchain development, here's a comparison be
   - [Cadence testing framework](https://cadence-lang.org/docs/testing-framework) enables native tests in Cadence.
   - [overflow](https://github.com/bjartek/overflow) for testing in Go.
   - [js-testing](https://github.com/onflow/flow-js-testing) for testing in JS.
+
+[Why Flow]: ../flow.md
+[EVM]: ../../evm/about
+[accounts on Ethereum]: https://ethereum.org/en/developers/docs/accounts/
```

Comment:
"accounts" needs to be capitalized

Reply:
Relative-style links, added comment to clarify!